Vagrantfile

```diff
@@ -0,0 +1,56 @@
+# Script for installation of pyenv
```

Comment:
If you're going to split the script up into smaller, helpfully named pieces, do you still need these comments?